packages/react/src/provider/use-open-feature-client-status.ts

```diff
@@ -1,15 +1,15 @@
 import { useEffect, useState } from 'react';
 import { useOpenFeatureClient } from './use-open-feature-client';
-import type { ProviderStatus } from '@openfeature/web-sdk';
-import { ProviderEvents } from '@openfeature/web-sdk';
+import { ProviderEvents, type ProviderStatus, } from '@openfeature/web-sdk';
+
 
 /**
  * Get the {@link ProviderStatus} for the OpenFeatureClient.
  * @returns {ProviderStatus} status of the client for this scope
  */
-export function useOpenFeatureClientStatus(): ProviderStatus {
+export function useOpenFeatureClientStatus() {
```

Comment:
I see [your comment](https://github.com/open-feature/js-sdk/pull/1082#discussion_r1835698885) about removing the return type, but I think it's worth keeping. Explicitly defining the return type ensures you don't accidentally return an unexpected value. In this case, we want to ensure we only return a `ProviderStatus` and not a generic `string` type.